documentation/docs/src/user-guide/ibc.md

```diff
@@ -0,0 +1,137 @@
+# Using IBC with Namada
+
+This document describes using the inter-blockchain communication (IBC) protocol with Namada. This documentation covers being able create connections through IBC as well as setting up local fractal instances of Namada for testing purposes.
+
+:::warning
+Warning! This is quite an advanced feature at the moment and is currently in development
+:::
+
+## Setting up
+
+The below is intended for those that wish to conduct IBC message transfer between two Namada chains. There is of course the cabablitiy to do this between any two IBC compatible chains (such as a cosmos chain). In this case, it is necessary to have a node of both the destination and the source chain running in order to make any package transfers. Below, we discuss setting up 2 Namada chains for this purpose.
+
+### Downloading the IBC-rs repo
+
+In order to create the relayer that is able to set up IBC connections, the following repo will need to be cloned. Then we enter the directory.
+
+```bash=
+git clone git@github.com:heliaxdev/ibc-rs.git && cd ibc-rs
+```
+
+
+
+
+### Running multiple instances of Namada
+
+In order to make IBC transfers across two Namada chains, one must be running two nodes of Namada at once. In order to do this, two CHAIN-IDs will need to be retrieved. In the case that there is only 1 testnet running, one may need to setup a local namada chain (which will be proof of authority since you will be the only one running it :P).
+
+Below, I discuss how to setup connections between two local namada testnets, and I append how to achieve ibc package transfers between arbitrary chain_ids afterwards.
+
+#### Setting up local Namada fractal instances
+In order to setup a local testnet, we have created an easy script that does this for you. The following command will setup two local testnets.
+
+In order for this to be completed, one must have cloned the namada directory and built the wasm scripts. This is achieved by the following command
+
+```bash=
+git clone 
+#Make sure you are in the home directory
+cd ~
+#Now clone
+git@github.com:anoma/namada.git && cd namada
+make build-wasm-scripts
+export NAMADA_DIR=${HOME}/namada
+```
+
+The second command requires being located in the achieved in the ibc-rs directory.
+
+
+
+```bash=
+cd ~/ibc-rs/
+./e2e/namada-test.sh ${NAMADA_DIR}
```

Comment:
The script runs also ibc-rs' E2E test and shut down Namada chains.
I made a new script `scripts/setup-namada.sh`.
Could you try it?